src/main/java/game/jdbc/picturepoker/PlayerDAO.java

```diff
@@ -11,9 +11,11 @@ public class PlayerDAO extends DataAccessObject<Player>{
     private static final String GET_PLAYER_BY_ID = "SELECT "
     + "p_id, p_name, passcode, dollars, first_places, second_places, third_places, "
     + "fourth_places, lifetime_tokens, tokens, bet, rounds_won FROM player WHERE p_id = ?";
-    private static final String CREATE_NEW_PLAYER = "INSERT INTO player (p_name, password) VALUES (?, ?)";
+    private static final String CREATE_NEW_PLAYER = "INSERT INTO player (p_name, passcode) VALUES (?, ?)";
+    private static final String CREATE_NEW_CARD = "INSERT INTO card (pid, hand_pos, suit) VALUES (?, ?, ?)";
```

Comment:
Also, I just noticed that card should probably be renamed player_card (as the table has been renamed). Again, I'm not as familiar with the new database - this could be completely correct structure.